src/macaron/policy_engine/examples/simple_example.dl

```diff
@@ -0,0 +1,8 @@
+/* Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved. */
+/* Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/. */
+
+
+#include "prelude.dl"
+
+Policy("auth-provenance", repositoryid, "") :-    check_passed(repositoryid, "mcn_provenance_level_three_1").
```

Comment:
```suggestion
Policy("auth-provenance", repositoryid, "") :- check_passed(repositoryid, "mcn_provenance_level_three_1").
```

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/46/commits/73c1c81b6bdb6c999112532dab89103685eca48b